### PR TITLE
Issues/561/wfdphotozreader

### PR DESCRIPTION
**Pull request by sschmidt23**
This PR addresses issue #561 to add a reader for run2.2i_dr6_v2 photo-z catalog for BPZ.  I did a quick check and things seem to work properly for grabbing both mag_i_cModel and photoz_mode, @yymao you might want to look at the parameters that I set in dc2_object_run2.2i_dr6_v2_with_addons.yaml and make sure that I made the proper choices.

### CODE REVIEW

**Review by yymao — CHANGES_REQUESTED** [review]
Thanks @sschmidt23. I made a suggestion to remove one unnecessary option. 

Do we expect people to access only the photo-z catalog itself, and not as part of the composite? If we do, that's fine.  If not, should we remove `photoz_dc2_object_run2.2i_dr6_v2.yaml`?

File: GCRCatalogs/catalog_configs/dc2_object_run2.2i_dr6_v2_with_addons.yaml [hunk 1]
```diff
@@ -18,6 +18,15 @@ catalogs:
     overwrite_quantities: false
     overwrite_attributes: false
     include_native_quantities: false
+  - subclass_name: dc2_photoz_parquet.DC2PhotozCatalog
+    base_dir: ^/DC2-prod/Run2.2i/addons/photoz/dr6_v2
+    filename_pattern: 'photoz_pdf_Run2.2i_dr6_tract_\d+\.parquet$'
+    as_object_addon: true
```
[the remark this answers is not in the document]
Comment (sschmidt23) [reply GCRCatalogs/catalog_configs/dc2_object_run2.2i_dr6_v2_with_addons.yaml]: removed

**Comment by sschmidt23** [comment]
That's a good point, I don't see many people accessing the photo-z's without any sort of ties to the photometry or positions, so I think we can remove the standalone catalog.  I deleted the yaml file from the commit.

**Review by yymao — APPROVED** [review]
Thanks, @sschmidt23

**Comment by heather999** [comment]
Just a quick question - are these PZ strictly for internal DESC use with DR6 or are these PZs also meant for public release?

**Comment by sschmidt23** [comment]
For now they are for internal release, but I made the reader in case they end up being the public release candidate.  Johann and Alex were chatting about a potential alternative candidate for public release candidate, but it's never actually been run before.  So, I wanted to prepare this set as a backup and make sure that it functioned.  It should be useful internally no matter what.

**Pull request merged**